manifests/supervisorcluster/1.27/cns-csi.yaml

```diff
@@ -449,6 +449,7 @@ data:
   "list-volumes": "true"
   "cnsmgr-suspend-create-volume": "true"
   "listview-tasks": "true"
+  "create-file-volume-with-topology": "false"
```

Comment:
why do you add this fss for supervisor yaml?

Reply:
Sorry I mistakenly put it in supervisor.